pkg/master/master.go

```diff
@@ -65,6 +65,7 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/registry/service"
 	ipallocator "github.com/GoogleCloudPlatform/kubernetes/pkg/registry/service/ipallocator"
 	etcdipallocator "github.com/GoogleCloudPlatform/kubernetes/pkg/registry/service/ipallocator/etcd"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/registry/service/portallocator"
```

Comment:
On first read - why does this follow a different pattern than ipallocator above it?